extensions/wikia/GlobalNavigation/scripts/GoogleSearchAPI.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * We AB test for wikia search (both within and all wikias) in Oasis with Google custom search result.
+ * window.wgGoogleSearchTest: a global variable set true in Optimizely for 50% of users who visit ja wikia.
+ * window.wgGoogleSearchParam: Search ID set in Optimizely.
+ */
+$(function () {
+	if ( window.wgGoogleSearchTest ) {
+		var searchForm = document.getElementById('searchForm'),
```

Comment:
So initially, I paster google JS snippet here that is why it is mix of plain JS and JQuery. I rewrote it to use JQuery for all. thanks.